doc/src/devenv/tipsandtricks.rst

```diff
@@ -110,3 +110,22 @@ Check the coding style of PHP code
   vm$> cd /var/www/lc/site
   vm$> ./vendor/bin/phpcs --runtime-set ignore_warnings_on_exit true -s \
     && echo You can commit: No errors found!
+
+Algolia indices configuration
+---------------------------
+For configuring algolia in development environment first specify the Apllication ID and Admin API Key in ``parameter.yml``
```

Comment:
"To configure Algolia in the development environment you need to specify the Application ID (site_algolia_app_id) and the Admin API Key (site_algolia_api_key) in the ansible/secrets/dev-vagrant.secrets.yml file."

Is it really called "Admin API Key"?

Reply:
Yes algolia use two keys one for inserting data into algolia(Admin API  Key)  another for searching functionality (Search-Only API Key)